<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smartवडा</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>sw_icon.png</normaloff>sw_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="signStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>690</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="signupPage">
     <widget class="QGroupBox" name="signupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>671</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background:transparent;
      border: 2px solid gray;
      border-radius: 10px;
color: rgb(23, 109, 2);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="address">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>290</y>
         <width>491</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>Address</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signup">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>580</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:blue;
color:white;</string>
       </property>
       <property name="text">
        <string>Sign Up</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="municipality">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>230</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Municipality/VDC</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="wardno">
       <property name="geometry">
        <rect>
         <x>411</x>
         <y>230</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>Ward No.</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="phoneNo">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>350</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Phone No,</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>410</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>E-mail ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mun_logo">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>470</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Municipality/VDC Logo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>520</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="confirmpassword">
       <property name="geometry">
        <rect>
         <x>311</x>
         <y>520</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Re-Type your Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="sw_logo">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>311</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
border-radius:0px;
border-image:url(:/pic/sw_logo.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="signin_2">
       <property name="geometry">
        <rect>
         <x>288</x>
         <y>630</y>
         <width>261</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>Already have account,Sign In Instead</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="signinPage">
     <widget class="QGroupBox" name="signinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>671</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1366</width>
        <height>768</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> background:transparent;
color: rgb(23, 109, 2);
      border: 2px solid gray;
      border-radius: 10px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="login_id">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>270</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>Id or Gmail</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_password">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>360</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signin">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>460</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid #8f8f91;
border-radius: 6px;
background:blue;
color:white;
</string>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
      <widget class="QLabel" name="sw_logo_">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>311</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
border-radius:0px;
border-image:url(:/pic/sw_logo.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="forgetPassword">
       <property name="geometry">
        <rect>
         <x>318</x>
         <y>540</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>Forgot your password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signup_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>540</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
border-left:50px;
</string>
       </property>
       <property name="text">
        <string>|Sign Up</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sw.qrc"/>
 </resources>
 <connections/>
</ui>
